jobs/kola-azure.Jenkinsfile

```diff
@@ -120,11 +122,16 @@ cosaPod(memory: "${cosa_memory_request_mb}Mi", kvm: false,
                     --container $azure_testing_storage_container        \
                     --blob-name $azure_image_name                       \
                     --file ${azure_image_filepath}
-                ore azure create-image --log-level=INFO                 \
+                # Create a fresh gallery image. Note that this will
+                # create the testing gallery if it does not exist, but
+                # it will be a no-op on gallery creation given the
+                # gallery exists in the specified region.
+                ore azure create-gallery-image --log-level=INFO
                     --azure-credentials \${AZURE_KOLA_TESTS_CONFIG}     \
-                    --resource-group $azure_testing_resource_group      \
                     --azure-location $region                            \
-                    --image-name $azure_image_name                      \
+                    --resource-group $azure_testing_resource_group      \
+                    --azure-hyper-v-generation "V2"                     \
```

Comment:
No longer needed?
```suggestion

```

Reply:
ah! thanks, copy/paste error when I was reworking the job. I'll remove it